e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -63,7 +63,13 @@ const QUESTION_LINE_CHART = {
   display: "line",
   query: {
     aggregation: [["count"]],
-    breakout: [["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }]],
+    breakout: [
```

Comment:
This change already has impact - all field references in queries created from dashboards are valid now.